### ISSUE TITLE
darknet compile error

### ISSUE THREAD

**Issue by JackeyGuo** [AlexeyAB/darknet#5370]
when i compile darknet in ubuntu16.04, opencv 3.4.3, cuda9.0 and cudnn 7.4.1, i get this error information. And my makefile setting is:
`GPU=1
CUDNN=1
CUDNN_HALF=0
OPENCV=1
AVX=0
OPENMP=0
LIBSO=0
ZED_CAMERA=0 # ZED SDK 3.0 and above
ZED_CAMERA_v2_8=0 # ZED SDK 2.X
USE_CPP=0
DEBUG=1`
`g++ -std=c++11 -std=c++11 -Iinclude/ -I3rdparty/stb/include -DDEBUG -DOPENCV `pkg-config --cflags opencv4 2> /dev/null || pkg-config --cflags opencv` -DGPU -I/usr/local/cuda-9.0/include/ -DCUDNN -Wall -Wfatal-errors -Wno-unused-result -Wno-unknown-pragmas -fPIC -DDEBUG -Ofast -DOPENCV -DGPU -DCUDNN -I/usr/local/cuda-9.0/include obj/image_opencv.o obj/http_stream.o obj/gemm.o obj/utils.o obj/dark_cuda.o obj/convolutional_layer.o obj/list.o obj/image.o obj/activations.o obj/im2col.o obj/col2im.o obj/blas.o obj/crop_layer.o obj/dropout_layer.o obj/maxpool_layer.o obj/softmax_layer.o obj/data.o obj/matrix.o obj/network.o obj/connected_layer.o obj/cost_layer.o obj/parser.o obj/option_list.o obj/darknet.o obj/detection_layer.o obj/captcha.o obj/route_layer.o obj/writing.o obj/box.o obj/nightmare.o obj/normalization_layer.o obj/avgpool_layer.o obj/coco.o obj/dice.o obj/yolo.o obj/detector.o obj/layer.o obj/compare.o obj/classifier.o obj/local_layer.o obj/swag.o obj/shortcut_layer.o obj/activation_layer.o obj/rnn_layer.o obj/gru_layer.o obj/rnn.o obj/rnn_vid.o obj/crnn_layer.o obj/demo.o obj/tag.o obj/cifar.o obj/go.o obj/batchnorm_layer.o obj/art.o obj/region_layer.o obj/reorg_layer.o obj/reorg_old_layer.o obj/super.o obj/voxel.o obj/tree.o obj/yolo_layer.o obj/gaussian_yolo_layer.o obj/upsample_layer.o obj/lstm_layer.o obj/conv_lstm_layer.o obj/scale_channels_layer.o obj/sam_layer.o obj/convolutional_kernels.o obj/activation_kernels.o obj/im2col_kernels.o obj/col2im_kernels.o obj/blas_kernels.o obj/crop_layer_kernels.o obj/dropout_layer_kernels.o obj/maxpool_layer_kernels.o obj/network_kernels.o obj/avgpool_layer_kernels.o -o darknet -lm -pthread `pkg-config --libs opencv4 2> /dev/null || pkg-config --libs opencv` -L/usr/local/cuda-9.0/lib64 -lcuda -lcudart -lcublas -lcurand -L/usr/local/cuda-9.0/lib64 -lcudnn -lstdc++ 
obj/classifier.o: In function `threat_classifier':
classifier.c:(.text+0x5119): undefined reference to `timersub'
collect2: error: ld returned 1 exit status
Makefile:155: recipe for target 'darknet' failed
make: *** [darknet] Error 1
`

**Comment by JackeyGuo** [comment]
And i tried set `OPENCV=0`, then the darknet can be compiled success! Can you recommend the version of opencv in ubuntu 16.04. Also, i have tried opencv version 4.3.0. thanks!

**Comment by AkkiSony** [comment]
@JackeyGuo Did you find any solution?
I have a similar error when I set GPU=1. 

![AWS-Make-File-Error](https://user-images.githubusercontent.com/85996229/122221533-a5230100-ceb1-11eb-9e70-c44a40abb453.PNG)

Please let me know if you have any solution. Thank you in advance.
